WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -259,9 +259,9 @@ private extension ReprintShippingLabelViewController {
     enum Localization {
         static let navigationBarTitle = NSLocalizedString("Print Shipping Label",
                                                           comment: "Navigation bar title to print a shipping label")
-        static let reprintButtonTitle = NSLocalizedString("Print Shipping Label",
+        static let printButtonTitle = NSLocalizedString("Print Shipping Label",
                                                           comment: "Button title to generate a shipping label document for printing")
-        static let paperSizeSelectorTitle = NSLocalizedString("Paper Size", comment: "Title of the paper size selector row for reprinting a shipping label")
+        static let paperSizeSelectorTitle = NSLocalizedString("Paper Size", comment: "Title of the paper size selector row for printing a shipping label")
```

Comment:
Should the header text be updated too (next line)? It says "print it again" and "reprinting"

Reply:
I left that as-is because that header text will only be used when we're reprinting an existing shipping label. I have customizations for this screen in the followup PR that will swap out the header text for use in the purchase flow (when the label is being printed for the first time).